### PR TITLE
RES-594: Move 'averageConvergencePoint' to 'L4L2Experiment' class

### PR DESCRIPTION
**Pull request by lscheinkman**
Moved `averageConvergencePoint` function to `L4L2Experiment` and changed the experiments to use the new method.

@subutai @ywcui1990 Please review

### CODE REVIEW

File: htmresearch/frameworks/layers/l2_l4_inference.py [hunk 1]
```diff
@@ -585,6 +586,59 @@ def getInferenceStats(self, experimentID=None):
...
 
+  def averageConvergencePoint(self, prefix, minOverlap, maxOverlap,
+                              settlingTime=1):
+
+    """
+    inferenceStats contains activity traces while the system visits each object.
+
+    Given the i'th object, inferenceStats[i] contains activity statistics for
+    each column for each region for the entire sequence of sensations.
+
```
Comment (subutai) [inline htmresearch/frameworks/layers/l2_l4_inference.py]: The above comments don't apply anymore do they?

File: htmresearch/frameworks/layers/l2_l4_inference.py [hunk 2]
```diff
@@ -841,6 +894,21 @@ def getDefaultFeedForwardSPParams(self, inputSize):
     }
 
 
+  def _locateConvergencePoint(self, stats, minOverlap, maxOverlap):
```
Comment (subutai) [inline htmresearch/frameworks/layers/l2_l4_inference.py]: Could be a static method.

File: projects/l2_pooling/multi_column_convergence.py [hunk 3]
```diff
@@ -23,11 +23,10 @@
 or adjust the confusion between objects.
 
 """
+import argparse
```
Comment (subutai) [inline projects/l2_pooling/multi_column_convergence.py]: Why?

**Comment by lscheinkman** [comment]
replaced by #803

**Pull request closed without merging**